dns/dnsperfgo/main.go

```diff
@@ -39,10 +41,10 @@ import (
 const clusterDomain = "cluster.local"
 
 type Config struct {
-	qps                                      int
-	testDuration, idleDuration, queryTimeout time.Duration
-	hostnameFile                             string
-	queryClusterNames, logQueries            bool
+	qps                                                 int
+	testDuration, idleDuration, queryTimeout            time.Duration
+	hostnameFile                                        string
+	queryClusterNames, logQueries, enableLatencyLogging bool
```

Comment:
[minor] prefer to have these on individual lines otherwise the gofmt will indent the type decl quite a bit to the right.